src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html, body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+
+/* ajustando tamanho */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 10px 20px;
+  background-color: #fff;
+  border-bottom: 1px solid #e0e0e0;
+  position: relative;
+  box-sizing: border-box;
+  height: 60px;
+}
+
+/* Logo */
+.logo img {
+  height: 40px;
+}
+
+/* Navegação */
+nav ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  margin-left: 20px;
+}
+
+nav ul li:first-child {
+  margin-left: 0;
+}
+
+nav ul li a {
+  text-decoration: none;
+  color: #000;
+  font-weight: bold;
+  position: relative;
+}
+
+nav ul li a:hover {
+  color: #00aaff;
+}
+
+/* ponteiro sobre o link ativo*/
+nav ul li a.is-active {
```

Comment:
As mentioned earlier, avoid using tag names for styling. It's better to style elements using classes.